src/pages/groupComparison/ClinicalData.tsx

```diff
@@ -267,17 +267,19 @@ export default class ClinicalData extends React.Component<
                         })
                 );
 
-                let clinicalData = await client.fetchClinicalDataUsingPOST({
-                    clinicalDataType: attribute.patientAttribute
-                        ? 'PATIENT'
-                        : 'SAMPLE',
-                    clinicalDataMultiStudyFilter: {
-                        attributeIds: [attribute.clinicalAttributeId],
-                        identifiers: attribute.patientAttribute
-                            ? patientidentifiers
-                            : sampleIdentifiers,
-                    },
-                });
+                let clinicalData = await client
+                    .handleErrorsGlobally()
```

Comment:
i'm torn whether to make global handling the default and allow people just to use the client directly (client.fetchClincal ...) or actually make them write out the method, which woudl force them to think about it.

Reply:
i think global handling/fatal error is a good default. its more unusual for an error to be tolerable

Reply:
as discussed, I think it should be mandatory to choose an error handling method

Reply:
I think it is okay to force the developer. We should think about every new API call we add to the application anyway.